backend/fms_core/template_importer/row_handlers/library_qc/library_qc.py

```diff
@@ -134,7 +138,7 @@ def process_row_inner(self, sample_container, measures, process, process_measure
 
         # set the library size on the library
         _, self.errors['library-size'], self.warnings['library-size'] = \
-           update_library(source_sample_obj, **{'library_size':library_size})
+            update_library(source_sample_obj, **{'library_size': library_size})
```

Comment:
We need to skip this update for pools or we will overwrite library_size with None for pools which is not the desired result (which is keeping the current value).